holoviews/plotting/plot.py

```diff
@@ -770,14 +797,17 @@ def _compute_group_range(cls, group, elements, ranges, framewise, robust, top_le
                         except:
                             factors = util.unique_array(values)
                     if dim_name not in group_ranges:
-                        group_ranges[dim_name] = {'data': [], 'hard': [], 'soft': []}
+                        group_ranges[dim_name] = {
+                            'id': [], 'data': [], 'hard': [], 'soft': []
+                        }
 
                     if factors is not None:
                         if 'factors' not in group_ranges[dim_name]:
                             group_ranges[dim_name]['factors'] = []
                         group_ranges[dim_name]['factors'].append(factors)
                     else:
                         group_ranges[dim_name]['data'].append(drange)
+                    group_ranges[dim_name]['id'].append(id(el))
```

Comment:
Guess this plotting class will accumulate ranges for all the elements it processes. Doubt it would cause much of a memory leak to worry about though...